packages/sanity/src/core/studio/tasks/components/TasksSidebar/TasksSidebar.tsx

```diff
@@ -0,0 +1,36 @@
+import {Card} from '@sanity/ui'
+import styled from 'styled-components'
+import {AnimatePresence, motion} from 'framer-motion'
+import {useTasksEnabled, useTasks} from '../../context'
+import {TasksSidebarHeader} from './TasksSidebarHeader'
+
+const SidebarRoot = styled(Card)`
+  height: 100%;
+  width: 360px;
+  margin-left: 4px;
+  box-shadow:
+    0px 6px 8px -4px rgba(134, 144, 160, 0.2),
```

Comment:
Do we want to use different `rgba` shadow values depending on the current scheme?

Reply:
I didn't wanted to get into that yet, as the designs are not final and this shadow is just a custom quick implementation copying what we have in the figma file. But for sure let's revisit when designs are final.